powersimdata/scenario/analyze.py

```diff
@@ -53,9 +53,7 @@ def refresh(self, scenario):
     def _set_allowed_state(self):
         """Sets allowed state."""
         if self._scenario_status == "extracted":
-            self.allowed = ["delete"]
-            if self._data_access.backup_root is not None:
-                self.allowed.append("move")
+            self.allowed = ["delete", "move"]
```

Comment:
`self.allowed == ["delete", "move"] if self._scenario_status == "extracted" else []` ?

Reply:
I tried but it still gets reformatted to the following:
```
        self.allowed = (
            ["delete", "move"] if self._scenario_status == "extracted" else []
        )

```

Reply:
So let's keep what we had then

Reply:
Will this work?
```python
self.allowed = ["delete", "move"]*(self._scenario_status == "extracted")
```

Reply:
Whoa, what kind of magic is this. It does work, but I'd rather keep it explicit. That's pretty neat though.